setup.py

```diff
@@ -25,13 +25,14 @@
         'numpy',
         'matplotlib<3.8.0',
         'tqdm',
+	'configobj',
         'mockito',
         'psutil',
         'pyopencl',
         'pyqt5',
-        'traitsui<8.0.0',
+        'traitsui',
         'vtk>=9.2.2',
-        'mayavi==4.8.1',
+        'mayavi>=4.8.1',
```

Comment:
We need to bump mayavi >= 4.8.2 for traitsui 8 compatibility.
https://docs.enthought.com/mayavi/mayavi/auto/changes.html